config/routes.php

```diff
@@ -125,6 +122,23 @@
     $routes->fallbacks(DashedRoute::class);
 });
 
+Router::prefix('api', function (RouteBuilder $routes) {
+    $routes->connect('/users/login', ['controller' => 'Users', 'action' => 'login'])
```

Comment:
you can add specific method so that you don't need setMethods
POST | $routes->post(...
GET   |  $routes->get(....